src/Control/LimitControl.php

```diff
@@ -116,6 +116,7 @@ protected function assemble()
             'class'   => 'autosubmit',
             'label'   => '#',
             'options' => $limits,
+            'title'   => sprintf('Change item count per page'),
```

Comment:
I think you should use the `t(…)` function for translation. You also don't need `sprintf()`, since you don't replace anything.